server/src/main/scala/FileRequester.scala

```diff
@@ -50,23 +70,47 @@ class FileRequester(mongoConfig: MongoConfig) {
 
   def create(x: FileDocument): Future[InsertOneResult] = {
     logger.info("Performing a create on {}", x.path)
+    x.version = Some(1)
```

Comment:
Maybe you can create a constant for the default version? So that if we need this value later we can simply use the constant